app/js/event/event.sort.js

```diff
@@ -32,7 +32,7 @@
           getDistanceFromLatLonInKm(Number(events[i].latitude),
                                     Number(events[i].longitude),
                                     Number(coordinates.latitude),
-                                    Number(coordinates.longitude));
+                                    Number(coordinates.longitude)).toFixed(2);
```

Comment:
`.toFixed()` is supposed to be used only in the filter. This factory should not know anything about displaying stuff, it just handles data.